fi-FI.Rmd

```diff
@@ -0,0 +1,115 @@
+
+Autamme mielellämme sinua saamaan tarvitsemasi tuen (voimme esimerkiksi auttaa ottamaan yhteyttä hotellin tai tilaisuuden turvapalveluihin, paikallisiin viranomaisiin tai sosiaalipäivystykseen, tai voimme järjestää tukihenkilön tai avustaa muilla tavoin varmistaaksemme, että tunnet olosi turvalliseksi tapahtuman ajan).
+
+Arvostamme osallistumistasi ja läsnäoloasi Bioconductor-tapahtumissa ja -yhteisössämme.
+
+#### _Välittäjähenkilö_
+
+Ymmärrämme, että saatat haluta, että Bioconductorista riippumaton henkilö valvoo tilanteen ratkaisua. Tämän mahdollistamiseksi käytännesääntökomitea nimittää Bioconductorista riippumattoman välittäjän toisesta avoimen lähdekoodin yhteisöstä. Vaikka hänellä ei ole valtuuksia asettaa, muuttaa tai kumota käytännesääntökomitean tekemiä päätöksiä, hän voi tukea yhteisön jäseniä tarjoamalla neuvontaa, välitystä, valmennusta ja ohjausta muihin resursseihin. Välittäjään voi olla suoraan yhteydessä sähköpostitse (katso alla), jotta prosessi on luottamuksellinen, puolueeton, epävirallinen ja itsenäinen.
+
+#### _Konfliktien käsittelyprosessi_
+
+Käytännesääntökomitea noudattaa vakiintunutta menettelyä konfliktien käsittelemiseksi ja ratkaisemiseksi.
+Tämä menettely on lyhyesti tiivistetty [täällä](https://bioconductor.github.io/CodeOfConduct/incident_response.html). Toimintatavat perustuvat Valerie Auroran ja Mary Gardinerin kirjoittamaan kirjaan "How to Respond to Code of Conduct Reports".
+
+
+### Käytännesääntökomitea
+
+* (Yhteispuheenjohtaja) Anna Quaglieri anna.quaglieri16@gmail.com (vuoteen 2024 asti) she/her [EN, IT]
+* (Yhteispuheenjohtaja) Gerry Tonkin-Hill g.tonkinhill@gmail.com (vuoteen 2025 asti) he/him [EN]
+* Martín Dávila-Cervantes aldntmi@gmail.com (vuoteen 2025 asti) he/him [EN, FR, ES] 
+* Federica Gazzelloni fede.gazzelloni@gmail.com (vuoteen 2025 asti) she/her [IT, EN]
+* Michael Love michaelisaiahlove@gmail.com (vuoteen 2024 asti) he/him [EN] (TAB)
+* Federico Marini fede.maro@gmail.com (vuoteen 2024 asti) he/him [EN, IT, DE]
+* Sowmya Parthiban sowmyaparthiban@gmail.com (vuoteen 2024 asti) she/her [EN, Tamil]
+* Shixiang Wang wangshx@shanghaitech.edu.cn (vuoteen 2025 asti) he/him [CN, EN]
+* Lluís Revilla lluis.revilla@gmail.com (vuoteen 2024 asti) he/him [CAT, ES, EN]
+* Kevin Rue-Albrecht kevinrue67@gmail.com  (vuoteen 2024 asti) he/him [FR, EN] (CAB)
+* Paulina Boadiwaa Mensah pabbyboadiwaa@gmail.com (vuoteen 2025 asti) she/her [EN, Akan]
+* Peace Sandy peacesandy04@gmail.com (vuoteen 2025 asti) she/her [EN]
+* Sinmisoluwa Adesanya adesanyasinmisoluwa@gmail.com (vuoteen 2025 asti) she/her [EN, Yoruba]
+* (Välittäjähenkilö) Michelle Savage michellesavage.galaxy@gmail.com (Galaxy project) she/her [ES, EN]
```

Comment:
Btw, I am not sure what to do with these pronouns. Translating them does not make sense because in Finnish we have "neutral" pronoun; everyone is just "hän/hänen". So should I remove them or keep in English?

Reply:
Good point. No one else raised this before so there is no guideline.I can raise the point with the community advisory board, to see if we can come up with a suggestion. At the moment, it's very much up to the person translating to set the standard :)